varcode/common.py

```diff
@@ -48,3 +49,10 @@ def wrapped_fn(*args, **kwargs):
             return memoized_values[key]
 
     return wrapped_fn
+
+
+def read_csv_as_df(*args, **kwargs):
```

Comment:
You make a fair point. That said, I couldn't find a good way to disable pylint's `E: 68,11: Instance of 'tuple' has no 'columns' member (no-member)`-type errors. So I'm just disabling them manually, for now.